pkgs/top-level/all-packages.nix

```diff
@@ -39882,6 +39882,8 @@ with pkgs;
     inherit (linuxPackages) nvidia_x11;
   };
 
+  normaliz = callPackage ../applications/science/math/normaliz { };
```

Comment:
Use the pkgs/by-name hierarchy

Reply:
I'm sorry, I'm not sure what you want me to do here. Do you want me to move the entire package to `pkgs/by-name/no/normaliz`? Is the old sorting by category completely discontinued or should I eg. symlink it to `pkgs/applications/science/math/`

Reply:
The old sorting-by-category is legacy and discouraged.
When possible, the new packages should be put on this new hierarchy.

Especifically, `pkgs/applications/science/math/normaliz/default.nix` should be moved to `pkgs/by-name/no/normaliz/package.nix`.

Reply:
Ah, okay. Didn't realise. Fixed it now